launch/launch/exit_handler.py

```diff
@@ -24,11 +24,13 @@ def __init__(self, launch_state, task_state):
         self.task_state = task_state
 
 
-def default_exit_handler(context):
+def default_exit_handler(context, ignore_returncode=False):
     """
     Trigger teardown of launch.
 
     Use the returncode of the task for the launch if the launch was not already tearing down.
+
+    @param ignore_returncode: If True, then ignore all process return codes.
```

Comment:
The exit handler is specific to a single process. `all` doesn't make sense here.